web/client/components/map/openlayers/plugins/WMSLayer.js

```diff
@@ -29,16 +31,63 @@ function getWMSURLs( urls ) {
     return urls.map((url) => url.split("\?")[0]);
 }
 
+function needProxy(uri) {
```

Comment:
Axios has almost identical function but implemented in request interceptor.
Where you want to implement this function?